packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -468,6 +468,13 @@ function getBookingData({
           if (val.responses) {
             const unwantedProps: string[] = [];
             legacyProps.forEach((legacyProp) => {
+              if (typeof val[legacyProp as keyof typeof val] !== "undefined") {
```

Comment:
Unrelated, gives a warning when passing `location` or `notes` as a falsy value (but still present). Just log as this would be a breaking change.